tests/tests_test_workflow/test_integ_workflow/integ_test/test_local_test_cluster.py

```diff
@@ -32,6 +33,9 @@ def setUp(self) -> None:
         self.dependency_installer = None
         self.test_recorder = None
 
+        ServiceOpenSearch.trans_stdout = ""
+        ServiceOpenSearch.trans_stderr = ""
```

Comment:
Same as above, why is it initialized to empty string?
If you have a new instance of the class, it should default to empty or None isnt it?

Reply:
That is another good point you mentioned. 
Since it is the test validation on local_test_cluster.py, I think I can actually add some test strings on them. I have added test string and updated the assertion expectation accordingly in the latest update. Hope that works better. Thanks!